tests/test_admin.py

```diff
@@ -131,12 +132,63 @@ def test_get_queryset_filters_by_content_id(self):
         )
 
 
+class MenuItemAdminVersionLocked(CMSTestCase, UsefulAssertsMixin):
+
+    def setUp(self):
+        # who is author..
+        self.menu_content = factories.MenuContentWithVersionFactory(version__state=DRAFT)
+        self.item = factories.ChildMenuItemFactory(parent=self.menu_content.root)
+
+        self.change_url = reverse(
+            "admin:djangocms_navigation_menuitem_change",
+            kwargs={"menu_content_id": self.menu_content.pk, "object_id": self.item.pk}
+        )
+        self.client.force_login(self.get_superuser())
+
+        # moving a node
+        menu_content = factories.MenuContentWithVersionFactory()
+        child = factories.ChildMenuItemFactory(parent=menu_content.root)
+        child_of_child = factories.ChildMenuItemFactory(parent=child)
+        self.move_url = reverse(
+            "admin:djangocms_navigation_menuitem_move_node", args=(menu_content.id,)
+        )
+        self.data = {
+            "node_id": child_of_child.pk,
+            "sibling_id": menu_content.root.pk,
+            "as_child": 1,
+        }
+
+    def test_visit_change_view_when_node_is_version_locked_fails(self):
+        """It fails as super user is not the person who created the version"""
```

Comment:
The docstrings should be on every test and should describes the business requirement with what and why. In this instance this could be: Visiting the change view for a menu item by a different user should be rejected, because the user is different to the locked to user.

This helps anyone making changes in the future who could break this test understand what it was trying to do, if it then needs to be reworked it can be as the developer should be informed of the requirement that the test is enforcing.